sagemaker-debugger/mnist_tensor_analysis/mnist_tensor_analysis.ipynb

```diff
@@ -2,7 +2,6 @@
  "cells": [
```

Comment:
<p>Why are we removing smedebug module install? Can we verify both recommended kernels in MEAD/Studio have this package preinstalled?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I moved that to the later part in this notebook, where the library is actually needed. Please check the blow parts.</p>